src/client/ocpp/json/JsonRestChargingStationClient.ts

```diff
@@ -95,6 +96,8 @@ export default class JsonRestChargingStationClient extends ChargingStationClient
       module: MODULE_NAME, method: 'onOpen',
       message: `Try to connect to '${this.serverURL}' ${Configuration.isCloudFoundry() ? ', CF Instance \'' + this.chargingStation.cfApplicationIDAndInstanceIndex + '\'' : ''}`
     });
+    Utils.isDevelopmentEnv() && console.log(`REST client try to connect to '${this.serverURL}' ${Configuration.isCloudFoundry() ? ', CF Instance \'' + this.chargingStation.cfApplicationIDAndInstanceIndex + '\'' : ''}`);
```

Comment:
If you want dev console log, you can enable it in the Logging class:

![image](https://user-images.githubusercontent.com/25266945/103819453-bea26000-506a-11eb-8b60-3bbaec5eac87.png)

And you can enable it here:

![image](https://user-images.githubusercontent.com/25266945/103819560-f5787600-506a-11eb-8b50-3a910a05a5d7.png)

Reply:
@jerome-benoit Have you removed the console logs and use the settings above?

Reply:
> @jerome-benoit Have you removed the console logs and use the settings above?

In the PR description: 

"Update 2:

Remove useless console logging (in WSClient it has to stay)."

Reply:
Ok thanks, overlooked it!